src/components/Search.js

```diff
@@ -1,22 +1,48 @@
-import React from 'react';
+import React from "react";
+import ButtonSearch from "./Button";
 
-const Search = props => (
-  <div className="search">
-    <div className="page-header">
-      <h4 className="text-left">Search Bookings</h4>
-    </div>
-    <div className="row search-wrapper">
-      <div className="col">
-        <div className="form-group search-box">
-          <label htmlFor="customerId">Customer Id</label>
-          <div className="search-row">
-            <input type="text" id="customerId" className="form-control" placeholder="Customer Id" />
-            <button className="btn btn-primary fn-submit-name">Search IDs</button>
+const Search = props => {
+  console.log(props);
+  return (
+    <div className="search">
+      <div className="page-header">
+        <h4 className="text-left">Search Bookings</h4>
+      </div>
+
+      <div className="row search-wrapper">
+        <div className="col">
+          <div className="form-group search-box">
+            <label htmlFor="customerId">Customer Id</label>
+            <div className="search-row">
+              <input
+                type="text"
+                id="customerId"
+                className="form-control"
+                placeholder="Customer Id"
+              />
+              <ButtonSearch search={props.search} buttonText="Search Id" />
+            </div>
           </div>
         </div>
       </div>
-    </div>
-  </div>
-);
 
+      <div className="row search-wrapper">
```

Comment:
This is not part of the homework, but there is some duplication here. How could we improve this?